### DISCUSSION TITLE
[feature request] Notifications about discussions in Slack

### DISCUSSION THREAD

**Discussion by erikr** [community/community#3091] [Discussions] [answered]
**Context:**
We recently enabled Discussions in [a repo](https://github.com/aguirre-lab/ml4c3) and want to get notifications about new Discussions or replies in a Slack channel.

**Current problem(s):**
In the Slack channel
```
/github subscribe org/repo-name discussions
```
results in
```
Uh oh! discussions is not a feature. Available features are:
issues, pulls, deployments, statuses, public, commits, releases, comments, branches, reviews, required_labels
```

**Steps to Reproduce:**
See syntax above.

**Expected behavior:**
The GitHub app supports notifications from Discussions.

**Comment by erikr** [comment, marked as answer]
My apologies, I should search for existing Discussions first! https://github.com/github/feedback/discussions/2844

I can't close this Discussion (that is also a requested feature https://github.com/github/feedback/discussions/20).

For cohesion, suggest all subsequent comments go in the existing discussion linked above instead of here.

**Reply by evi-liu** [reply]
Thank you for linking to the relevant discussions!
